src/reanimated2/plugin.test.js

```diff
@@ -0,0 +1,53 @@
+import plugin from '../../plugin';
+import { transform } from '@babel/core';
+import { format } from 'prettier';
+
+function runPlugin(input) {
+  return transform(input, {
+    filename: 'someFile.js',
+    plugins: [plugin],
+  });
+}
+
+function prettierOutput(input) {
```

Comment:
opted for prettifying the output before snapshots for readability. Otherwise we're going to be diffing against a loooong one-liner.

Reply:
`compact: false` in transform options won't help with that? It should prevent babel from removing newlines

Reply:
ah nice @jakub-gonet! That's the option I was looking for but couldn't find. Thanks!

Added a commit with `compact: false`